_config/security.yml

```diff
@@ -31,5 +31,8 @@ SilverStripe\Core\Injector\Injector:
     properties:
       Authenticators:
         default: %$SilverStripe\Security\MemberAuthenticator\MemberAuthenticator
+  SilverStripe\Security\CMSSecurity:
```

Comment:
For the defaults, I think so. Because we don't want the standard `Security` to have the `CMS` login option, as it is unneeded in this case.

We _can_ have multiple per `Security`-class easily, I can confirm this works (though my tests are failing because it's actual two step now, instead of "try in a single step")